src/server/page_server.cpp

```diff
@@ -78,6 +83,22 @@ page_server::receive_log_prior_list (cubpacking::unpacker &upk)
   log_Gl.m_prior_recver.push_message (std::move (message));
 }
 
+void
+page_server::receive_log_page_fetch (cubpacking::unpacker &upk)
+{
+        if (prm_get_bool_value (PRM_ID_ER_LOG_READ_LOG_PAGE))
+        {
+                LOG_PAGEID pageid;
+                std::string message;
+
+                upk.unpack_string(message);
+                memcpy(&pageid, message.c_str(), sizeof(pageid));

```

Comment:
Use std:: namespace for standard functions.
```suggestion
                std::memcpy(&pageid, message.c_str(), sizeof(pageid));

```